repo/packages/A/arangodb/1/mantl.json

```diff
@@ -0,0 +1,13 @@
+{
+  "mesos": {
+    "master": "zk://zookeeper.service.consul:2181/mesos"
+  },
+  "arangodb": {
+    "id": "arangodb",
```

Comment:
I think we need to add an `arangodb.zk` property here. In https://github.com/mesosphere/universe/blob/version-1.x/repo/packages/A/arangodb/1/config.json, the default is `zk://master.mesos:2181/arangodb`. So, we probably need to set it to `zk://zookeeper.service.consul:2181/arangodb`